shared/js/otSdkDriver.js

```diff
@@ -560,8 +563,6 @@ loop.OTSdkDriver = (function() {
      * https://tokbox.com/opentok/libraries/client/js/reference/StreamEvent.html
      */
     _onRemoteStreamCreated: function(event) {
-      this._notifyMetricsEvent("Session.streamCreated");
```

Comment:
The old one wasn't very useful as it actually had two meanings. However, it made me think a bit, and from a stats analysis perspective, it'll be easier if the name changes, and we just drop the old one - hence I've changed `Session.streamCreated` to `Session.av.streamCreated` for better clarity as to when it kicks in.